api/src/main/java/org/openmrs/api/UserService.java

```diff
@@ -331,6 +331,18 @@ public interface UserService extends OpenmrsService {
 	 */
 	@Logging(ignoredArgumentIndexes = { 0, 1 })
 	public void changePassword(String oldPassword, String newPassword) throws APIException;
+
+	/**
+	 * Changes password of {@link User} passed in
+	 * @param user user whose password is to be changed
+	 * @param newPassword new password to set
+	 * @throws APIException
+	 * <strong>Should</strong> update password of given user when logged in user has edit users password privilege
+	 * <strong>Should</strong> not update password of given user when logged in user does not have edit users password privilege
+	 */
+	@Deprecated
```

Comment:
I flagged this as deprecated, but we could debate whether we really want to deprecate it.
Also we could potentially decide to deprecate it i the 2.5.x and 2.6.x branches you remove it in the 2.7.x

Reply:
I would fully use the deprecation guidelines here: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Deprecation 😊

Reply:
Upon further reflection/discussion, I don't think we want to deprecate this method at all.